### PR TITLE
remove redundant class and add copied tooltip/open in new tab

### PR DESCRIPTION
**Pull request by sanskar-mk2**
image
image

### CODE REVIEW

**Comment by sanskar-mk2** [comment]
@axeloz Any changes needed?

**Comment by axeloz** [comment]
sorry @sanskar-mk2, I'm completely under water these days. 
I will review this soon. Thanks for the PR 👍🏻

File: resources/views/components/newtab.blade.php [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<svg class="w-6 h-6 fill-primary"
```
Comment (axeloz) [inline resources/views/components/newtab.blade.php:1]: Can you please reduce the size to w-5 and h-5

File: resources/views/upload.blade.php [hunk 2]
```diff
@@ -629,28 +656,36 @@ class="relative flex items-center leading-5 list-inside even:bg-gray-50 rounded
...
-						<div class="flex flex-wrap items-center">
+						<div class="flex flex-wrap items-center relative">
 							<div class="w-1/3 text-right px-2">
 								@lang('app.preview-link')
 							</div>
-							<div class="w-2/3 shadow">
-								<input x-model="bundle.preview_link" class="w-full bg-transparent text-slate-700 h-8 px-2 py-1 rounded-none border border-primary-superlight outline-none" type="text" readonly x-on:click="selectCopy($el)" />
+							<x-tooltip x-show="copynotify.preview" x-on:click.away="copynotify.preview = false" />
+							<div class="w-2/3 shadow flex items-center">
+								<input id="copy-preview" x-model="bundle.preview_link" class="w-full bg-transparent text-slate-700 h-8 px-2 py-1 rounded-none border border-primary-superlight outline-none" type="text" readonly x-on:click="selectCopy($el)" />
```
Comment (axeloz) [inline resources/views/upload.blade.php:671]: - Missing translation for "Open in new tab"
- Move the input field border to include the icon: `border` is moved to the parent div

My proposal: 
```
<div class="w-2/3 shadow flex items-center border border-primary-superlight">
   <input id="copy-preview" x-model="bundle.preview_link" class="w-full bg-transparent text-slate-700 h-8 px-2 py-1 rounded-none outline-none" type="text" readonly x-on:click="selectCopy($el)" />
   <a class="p-1" title="@lang('app.open-in-new-tab')" :href="bundle.preview_link" target="_blank">
        <x-newtab />
    </a>
</div>
```

**Review by axeloz — COMMENTED** [review]
Thanks for your PR. LGTM, just a tiny UI improvement.

**Pull request merged**